package.json

```
@@ -92,6 +92,7 @@
     "rimraf": "3.0.2",
     "ts-jest": "28.0.8",
     "ts-node": "10.9.1",
+    "ts-toolbelt": "^9.6.0",
```

issue: pin the version
```suggestion
    "ts-toolbelt": "^9.6.0",
```
run `yarn install` afterwards to update the lockfile